hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -179,16 +193,21 @@ withTUITest region action = do
       , getPicture
       , shouldRender = \expected -> do
           bytes <- getPicture
-          -- Split at '\ESC' (27) and drop until 'm' (109)
-          let unescaped = BS.concat $ BS.drop 1 . BS.dropWhile (/= 109) <$> BS.split 27 bytes
-          unless (expected `BS.isInfixOf` unescaped) $
-            failure $
-              "Expected bytes not found in frame: "
-                <> decodeUtf8 expected
-                <> "\n"
-                <> decodeUtf8 bytes -- use colored frame (= with escape codes)
+          shouldRenderIf unless expected bytes
+      , shouldNotRender = \expected -> do
+          bytes <- getPicture
+          shouldRenderIf when expected bytes
```

Comment:
The error message is wrong in this case. I would suggest to factor out a function `findBytes` which does the escaping etc. and keep the the `when (isJust . findBytes ..) $ failure "Expected to NOT find bytes in frame..."`

Reply:
you are right! good catch!
On my way! 🏃